RecoParticleFlow/PFProducer/plugins/BuildFile.xml

```diff
@@ -79,5 +79,8 @@
   <use name="RecoParticleFlow/PFProducer"/>
   <use name="RecoParticleFlow/PFTracking"/>
   <use name="RecoEcal/EgammaCoreTools"/>
+  <use name="PhysicsTools/TensorFlow"/>
+  <use name="HeterogeneousCore/SonicCore"/>
+  <use name="HeterogeneousCore/SonicTriton"/>
```

Comment:
given the complexity of the dependencies, it may be practical to have a separate package

Reply:
Fair point. How about RecoParticleFlow/MLPFProducer? A new cms-data repo will need to be created, I wonder if there are other things to keep in mind when creating a new package in CMSSW?

Reply:
New packages should also be added to appropriate categories in https://github.com/cms-sw/cms-bot/. So far, we don't have a clear guideline about how to organize and isolate various dependencies...